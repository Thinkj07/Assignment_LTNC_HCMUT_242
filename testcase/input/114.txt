INSERT str string
ASSIGN str 'abc cd'