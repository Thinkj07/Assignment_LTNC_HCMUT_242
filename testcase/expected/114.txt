Invalid: ASSIGN str 'abc cd'